### PR TITLE
method name updated

### PR DESCRIPTION
**Pull request by imranrana07**
method name updated to getAuthor()

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (hasancse91) [inline .idea/vcs.xml:1]: not necessary to add this on Git. Please update your gitignore file.

**Review by hasancse91 — CHANGES_REQUESTED** [review]
Looks good. nice job!
Please review the comment.

**Review by hasancse91 — APPROVED** [review]
Looks good at me.

**Review by hasancse91 — CHANGES_REQUESTED** [review]
Looks good to me.

**Review by hasancse91 — APPROVED** [review]
Looks good to me.

**Comment by imranrana07** [comment]
method name and git ignore issue solved

**Pull request merged**